test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+﻿Describe "DeserializedMethods" -Tags "CI" {    
+    It "Deserialized objects shouldn't ever have any methods (unless they are primitive known types)" {
+        $a = new-object collections.arraylist
```

Comment:
I think we should use the `[type]::new()` syntax in tests other than testing `New-Object` because it is much faster.